<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditPriceCountDialog</class>
 <widget class="QDialog" name="EditPriceCountDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>316</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Count and Price</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="labelAlignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Count</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Price</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="doublePrice">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximum">
        <double>1000000000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Discount Formula</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineDiscount"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Discount</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="labelDiscount">
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Final Price</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="labelFinal">
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="labelMasterPrice">
       <property name="styleSheet">
        <string notr="true">color: red; font-style: italic;</string>
       </property>
       <property name="text">
        <string>Master Price : 0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="labelMasterDiscount">
       <property name="styleSheet">
        <string notr="true">color: red; font-style: italic;</string>
       </property>
       <property name="text">
        <string>Master Discount : 0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Note</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QPlainTextEdit" name="plainNote">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>75</height>
        </size>
       </property>
       <property name="tabChangesFocus">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>Input note if any</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="LibGUI::MoneyLineEdit" name="doubleCount"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushSave">
       <property name="text">
        <string>Save</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LibGUI::MoneyLineEdit</class>
   <extends>QLineEdit</extends>
   <header>moneylineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>doubleCount</tabstop>
  <tabstop>doublePrice</tabstop>
  <tabstop>lineDiscount</tabstop>
  <tabstop>plainNote</tabstop>
  <tabstop>pushCancel</tabstop>
  <tabstop>pushSave</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushCancel</sender>
   <signal>pressed()</signal>
   <receiver>EditPriceCountDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
